static/panes/compiler.js

```diff
@@ -1411,7 +1411,7 @@ Compiler.prototype.onCompileResponse = function (request, result, cached) {
 Compiler.prototype.postCompilationResult = function (request, result, wasCmake) {
     if (result.popularArguments) {
         this.handlePopularArgumentsResult(result.popularArguments);
-    } else {
+    } else if (this.compiler != null) {
```

Comment:
mayybe just "this.compiler" to be the opposite of line 2493 (I'm not sure of the subtleties between all the various "nullish or undefinedish" checks)